com.unity.build-report-inspector/Editor/BuildReportInspector/Mobile/PostBuildSetup.cs

```diff
@@ -0,0 +1,49 @@
+#if UNITY_ANDROID || UNITY_IOS
+using System.IO;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEditor.Callbacks;
+
+namespace Unity.BuildReportInspector.Mobile
+{
+    internal class PostBuildSetup : IPostprocessBuildWithReport, IPreprocessBuildWithReport
+    {
+        private const string k_GuidFileName = "UnityBuildGuid.txt";
+        private static string s_LastBuildGuid;
+        
+        public int callbackOrder { get; }
+        public void OnPreprocessBuild(BuildReport report)
+        {
+            s_LastBuildGuid = null;
+        }
+
+        public void OnPostprocessBuild(BuildReport report)
+        {
+            if (report.summary.result == BuildResult.Failed || 
+                report.summary.result == BuildResult.Cancelled)
+                return;
+                    
+            // Save the guid for BuildPostProcess callback.
+            s_LastBuildGuid = report.summary.guid.ToString();
```

Comment:
Move this after the `if` statement below?

Reply:
This is not needed for iOS, that's why I kept it above the if